lib/grape/validations/validators/default.rb

```diff
@@ -6,8 +6,30 @@ def initialize(attrs, options, required, scope)
         super
       end
 
```

Comment:
In Ruby `private def` does nothing AFAIK, it treats it as two separate things. Write:

```ruby
private

def duplicatable?(...)
...
end

def duplicate(...)
end
```